app/views/templates/paymentsHistoryCharge.scala.html

```diff
@@ -0,0 +1,77 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.templates.PaymentsHistoryChargeHelper
+@import views.html.templates.formatters.dates._
+@import views.html.templates.formatters.money._
+@import views.html.templates.payments._
+@import models.viewModels.PaymentsHistoryModel
+@import java.time.LocalDate
+
+@(transaction: PaymentsHistoryModel)(implicit messages: Messages)
+  
+@getDescriptionString(chargeTypeDescription: String, firstDate: Option[LocalDate], secondDate: Option[LocalDate]) = @{
+  (firstDate, secondDate) match {
+    case (Some(fd), Some(sd)) => {
+      messages(
+        s"$chargeTypeDescription",
+        s"${fd.getDayOfMonth} ${fd.getMonth.toString.head}${fd.getMonth.toString.toLowerCase.substring(1, 3)}",
+        s"${sd.getDayOfMonth} ${sd.getMonth.toString.head}${sd.getMonth.toString.toLowerCase.substring(1, 3)} ${sd.getYear}"
+      )
+    }
+
+    case (Some(fd), None) => {
```

Comment:
To increase the coverage can you add tests for the first or second date being a None please. I'm not sure though that we can get a charge with only one date though so those cases may not be required